meteor/sofie-server/data-access/repositories/mongo/mongo-rundown-playlist-repository.ts

```diff
@@ -0,0 +1,72 @@
+import { RundownRepository } from '../interfaces/rundown-repository'
+import { BaseMongoRepository } from './base-mongo-repository'
+import { Rundown } from '../../../model/entities/rundown'
+import { MongoDatabase } from './mongo-database'
+import { MongoEntityConverter, MongoRundownPlaylist } from './mongo-entity-converter'
+import { BasicRundown } from '../../../model/entities/basic-rundown'
+
+const RUNDOWN_PLAYLIST_COLLECTION_NAME: string = 'rundownPlaylists'
+
+export class MongoRundownPlaylistRepository extends BaseMongoRepository implements RundownRepository {
+	constructor(
+		mongoDatabase: MongoDatabase,
+		mongoEntityConverter: MongoEntityConverter,
+		private rundownRepository: RundownRepository
+	) {
+		super(mongoDatabase, mongoEntityConverter)
+	}
+
+	protected getCollectionName(): string {
+		return RUNDOWN_PLAYLIST_COLLECTION_NAME
+	}
+
+	async getBasicRundowns(): Promise<BasicRundown[]> {
+		this.assertDatabaseConnection('getRundownPlaylists')
```

Comment:
The input to the method has not been updated, when you renamed the method.

Would highly suggest it gets changed to `this.assertDatabaseConnection(getBasicRundowns.name)`

Doing it like this, will mean that future renaming will fix name used in the call too.

Reply:
I've actually already done that for the incoming commit, but very true :-)